packages/client/hmi-client/src/workflow/ops/model-config/model-config-operation.ts

```diff
@@ -24,7 +25,7 @@ export const ModelConfigOperation: Operation = {
 	displayName: 'Configure model',
 	description: 'Create model configurations.',
 	isRunnable: true,
-	inputs: [{ type: 'modelId' }],
+	inputs: [{ type: 'modelId' }, { type: 'documentId', label: 'Text' }],
```

Comment:
Add `(optional)` to the Text label

![image](https://github.com/DARPA-ASKEM/terarium/assets/17088680/ccd845cf-50a9-46a7-9dec-a8eaa7b6712e)

Reply:
^ No longer required. You can mark this input as isOptional and it will do this for you

Reply:
done